docs/wiki/framework/flash-framework.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: wiki
+title: Flashbang Framework
+description: Explains how to set-up flashbangs, particularly multi-bangs.
+group: framework
+parent: wiki
+order: 7
+mod: ace
+version:
+  major: 3
+  minor: 13
+  patch: 0
+---
+
+## 1. Overview
+
+ACE provides a simple interface for creating flashbang grenades and specifying their properties.
+
+
+## 2. Config Values
+
+There is only one config entry that is specific to ACE flashbangs: `ace_grenades_flashbang`. All successive values can be left undefined and they will be given the defaults shown in the first example below.
```

Comment:
As commy2 pointed out on Slack,

> commy2 [9:48 PM]
> For this to work you have to make sure that lifeTime is sufficiently longer than fuzeTime plus amount-1)*interval plus randomness.
> Otherwise the last explosions get lost.
> I think they spawn at 0,0,0 if the grenade projectiles becomes null.